codex/rest/api.nim

```diff
@@ -177,6 +174,47 @@ proc initRestApi*(node: CodexNodeRef, conf: CodexConf): RestRouter =
         if not stream.isNil:
           await stream.close()
 
+proc initRestApi*(node: CodexNodeRef, conf: CodexConf): RestRouter =
+  var router = RestRouter.init(validate)
+
+  initContentApi(node, router)
```

Comment:
You're right it doesn't make it more consistent. I think I'll group the other endpoints into their own init procs too. This keeps them together and organized, instead of appearing in one long function in somewhat random order. This might make the changeset-view a bit annoying, so please tell me what you think of the resulting code. The only call that's left is `/connect/peerid`, it doesn't seem to fit into any group yet. But if we get more API calls in the category of peer-networking, I'm sure this one will feel right at home!

Reply:
I'd leave this sort of cleanup for a follow up PR, lets just move the content api back into the one big proc for now and we can indulge in bikeshedding in the followups. It will also keep this PR diffs more readable/approachable.